src/lexedata/importer/fromexcel.py

```diff
@@ -323,9 +320,9 @@ def parse_cells(self, sheet: openpyxl.worksheet.worksheet.Worksheet) -> None:
                         form[c_f_id] = "{:}_{:}".format(
                             form[c_f_language], row_object[c_r_id]
                         )
-                    candidate_forms = iter(
-                        self.db.find_db_candidates(form, self.check_for_match)
-                    )
+                    candidate_forms = self.db.find_db_candidates(form, self.check_for_match)
+
```

Comment:
Remove this empty line

Reply:
Is this empty line against black? Just curious.

Reply:
No, it's just semantically strange, because empty lines are handy to structure code, and this adds a gap between the candidate forms search and the subsequent manipulation of these candidates.